plugins/georeplication/transactions.go

```diff
@@ -255,7 +255,7 @@ func configFileGenerate(session *georepapi.GeorepSession) error {
 	}
 
 	// Remote host and UUID details
-	var remote []string
+	var remote = make([]string, 0, len(session.RemoteHosts))
```

Comment:
Again, premature optimization. Seems like you are too much focused on cleanup even when it's not necessarry

Reply:
I think this is necessary to have it. leaving as it is